src/bci_build/package/389-ds/README.md.j2

```diff
@@ -0,0 +1,67 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+[389 Directory Server](https://www.port389.org/) is a highly usable, fully featured, reliable and secure LDAP server implementation.
+
+## Usage
+
+By default, the image launches 389 Directory Server with the same configuration that comes with the SUSE Linux Enterprise Server, with one exception, a pre-configured Name Service Switch (NSS) configuration file (`/etc/nsswitch.conf`) in included.
+
+```ShellSession
+$ podman run -it --rm -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Volumes
+
+The database is stored in the directory `/data`. A new empty database will be created on container startup unless an existing database is already present in `/data`.
```

Comment:
Does this mean that `/data/` is a volume?

Reply:
yes, `/data` is a volume. I guess this sentence is a bit vague right? It does not explicitly mention that it is a volume.

Reply:
> I guess this sentence is a bit vague right? It does not explicitly mention that it is a volume.

It is not 100% clear that `/data/` is a volume. I'd rather mention it to be safe.

Reply:
Yes, /data is always a volume.

Reply:
added